.env

```diff
@@ -0,0 +1,7 @@
+# Provides additional granular control of local development behaviour (primarily aimed at UI development)
+# Copy this file to `.env.local` and tweak any settings as you require for your workflow
+# Be careful in this file - any settings in here will be those used to actually compile the UI code
+
+# Should local UI testing fail when console.error's etc. occur?
+# Set to any value to enable
+GRAFANA_TESTS_FAIL_ON_CONSOLE=
```

Comment:
Prefixed the name here, but any suggestions are welcomed. A predictable pattern would be good, if a little verbose

Reply:
Does the backend pick this file up? There is also the `GF_<SectionName>_<KeyName>` [convention](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables) which overrides configs to bear in mind. 

I'm slightly concerned that whilst this might make sense to a few it will create confusion for developers.